x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_essentials_license/rule_import_export/import_rules.ts

```diff
@@ -100,7 +101,7 @@ export default ({ getService }: FtrProviderContext): void => {
 
         const bodyToCompare = removeServerGeneratedProperties(body);
         expect(bodyToCompare).to.eql({
-          ...getSimpleRuleOutput('rule-1', false),
+          ...getSimpleRuleOutput('rule-1', false, ELASTICSEARCH_USERNAME),
```

Comment:
Nit: There is an util fn called `updateUsername` to avoid sending the default params for the rule output methods
```   
        const expectedRule = updateUsername(getSimpleRuleOutput(), ELASTICSEARCH_USERNAME);
```

Reply:
Updated to use! I'm going to leave the change to the utils of allowing user to be defined as it now doesn't make sense to hard code it to `elastic`.